blockchain/binance/explorer/client.go

```diff
@@ -0,0 +1,31 @@
+package explorer
+
+import (
+	"net/url"
+	"strconv"
+
+	"github.com/trustwallet/go-libs/client"
+)
+
+// URL: https://explorer.binance.org
```

Comment:
Same as about, use description instead of exposing URL. Even if the URL is public